libs/designsystem/src/lib/components/progress-circle/progress-circle.component.spec.ts

```diff
@@ -43,24 +44,44 @@ describe('ProgressCircleComponent', () => {
     });
 
     describe('diameter', () => {
-      it('should default to md (56px)', () => {
-        spectator.detectChanges();
-        expect(spectator.component.diameter).toBe(ProgressCircleComponent.DIAMETER_MAP.md);
+      it('should default to md', () => {
+        expect(spectator.component.diameter).toBe(56);
```

Comment:
Consider changing to 
```suggestion
        expect(spectator.component.diameter).toBe('md');
```